tests/scripts/start-cypress-tests

```diff
@@ -15,6 +15,7 @@ docker run --init -v $PWD:/workdir -w /workdir              \
     -e RANCHER_PASSWORD=$RANCHER_PASSWORD                   \
     -e RANCHER_URL=$RANCHER_URL                             \
     -e RANCHER_USER=$RANCHER_USER                           \
+    -e PROXY=$PROXY                                         \
```

Comment:
please decide whether we need it in cypress

Reply:
Removed.